docs/concept-exercises.md

```diff
@@ -0,0 +1,92 @@
+# Concept Exercises
+
+Concept Exercises replace V2's Core Exercises. They are exercises designed to teach specific (programming) concepts.
+
+## What do we mean by concepts?
+
+By concepts we mean things that a programmer would need to understand to be fluent in a language. We care specifically about how languages are **different**. What do I need to learn differently about numbers in Haskell to numbers in Ruby to be able to work with numbers in those languages. Two questions that we have felt useful to ask to establish this are:
+
+- If someone learnt Ruby, and someone learnt Haskell, what are the things that the two people learnt that are different?
+- If a Ruby programmer learnt Haskell, what new concepts would they have to learn, what knowledge would they have to disregard, and what syntax would they have to remap?
+
+By teaching concepts we aim to teach fluency.
+
+## What do we mean by "fluency?"
+
+By "Fluency", we mean: Do you **get** that language? Can you reason in that language? Do you write that language like a "native" programmer writes in it? Fluency is the ability to express oneself articulately in a particular language.
+
+"Fluency" is different to "Proficiency", where we use proficiency to mean: Can you write programs in the language (e.g. can you nav stdlib, docs, compose complex code structures).
+
+Exercism focuses on teaching Fluency not Proficiency. We aim to teach people to understand what a makes a language unique and how experienced programmers in that language would reason about - and solve - problems.
+
+## How are Concept Exercises designed and structured?
+
+Concept Exercises must have the following characteristics:
+
+- Each one has a clear learning goal.
+- They are language-specific, not generic.
+- Stubs/boilerplate are used to avoid the student having to learn/write unnecessary code on exercises.
+
+Concept Exercises are **not** mentored. When a user submits a submission that gets the tests passing for a Concept Exercise, we check for an Analyzer or Representer to give feedback. If none is found, then the solution is approved. This shifts the burden of teaching to the exercise, which must provide a clear pathway to learning the concept that needs learning.
+
+Exercises are unlocked based on concepts taught and learnt. Each Concept Exercise must teach one or more concepts. It may also have prerequisites on Concepts, which means it will not be unlocked until Concept Exercises teaching those prerequisite concepts have been completed.
```

Comment:
```suggestion
Exercises are unlocked based on concepts taught and learnt. Each Concept Exercise must teach one or more concepts. It may also have prerequisites on concepts, which means it will not be unlocked until Concept Exercises teaching those prerequisite concepts have been completed.
```

The lowercase form is used later in the sentence too.